cmake/libs/libfaiss.cmake

```diff
@@ -48,7 +48,7 @@ endif()
 
 if(__AARCH64)
   set(UTILS_SRC src/simd/hook.cc src/simd/distances_ref.cc
-                src/simd/distances_neon.cc)
+                src/simd/distances_neon.cc src/simd/distances_sve.cc)
```

Comment:
I believe that this is not sufficient.
Knowhere is designed as a library that picks function pointers according to CPU capabilities, detected upon the start.
For example, SSE / AVX2 / AVX512 code files have different corresponding compile options
https://github.com/zilliztech/knowhere/blob/1cb99379b66d279dd8060f92d7d07491d6bd409b/cmake/libs/libfaiss.cmake#L37-L40

So, it seems to be logical that the new SVE code should also contain some form of different flags, such as `-march=armv8-a+sve` for `distances_sve.cc`. And I don't believe that I see this.

Reply:
Thank you for your valuable feedback @alexanderguzhva 

We considered two approaches: first, to keep Neon as the default until all FP32 functions for SVE have been fully implemented. Based on this, we decided to proceed with this approach.

Your approach is absolutely correct, and we agree that the Neon fallback for non-SVE functions can be effectively handled using hook.cc.

We have incorporated these changes into our solution.